spec/exporters/formatters/utaac_decision_indexable_formatter_spec.rb

```diff
@@ -2,7 +2,6 @@
 require "formatters/aaib_report_indexable_formatter"
```

Comment:
I know this wasn't introduced in this PR, but could we fix this require to refer to the correct source file?